### DISCUSSION TITLE
[Postal] Subdomain issue

### DISCUSSION THREAD

**Discussion by Truongwinner** [postalserver/postal#1579] [Help with using Postal]
Hi support team,

After installing, I can only access the Postal Server as the URL: '**http://mail.mydomain.com:5000**' instead of '**https://mail.mydomain.com**'

Could you please help me fix it or let me know what's wrong with my setting?

Thanks a lot,

Truong

**Comment by willpower232** [comment]
It sounds like you are missing something like Caddy or nginx to handle the HTTPS and proxy to port 5000.

Also, it sounds like you don't have a firewall in place as port 5000 should not be accessible externally.

**Reply by Truongwinner** [reply]
OK, noted. 
I will check it. Thanks for your reply!
